.gherkin-lintrc

```diff
@@ -0,0 +1,55 @@
+{
+  "no-files-without-scenarios": "on",
+  "no-unnamed-features": "on",
+  "no-unnamed-scenarios": "on",
+  "no-dupe-scenario-names": [
+    "on",
+    "in-feature"
+  ],
+  "no-dupe-feature-names": "on",
+  "no-partially-commented-tag-lines": "on",
+  "indentation": [
+    "on",
+    {
+      "Feature": 0,
+      "Background": 2,
+      "Scenario": 2,
+      "Step": 4
+    }
+  ],
+  "no-trailing-spaces": "on",
+  "new-line-at-eof": [
+    "on",
+    "yes"
+  ],
+  "no-multiple-empty-lines": "on",
+  "no-empty-file": "on",
+  "no-scenario-outlines-without-examples": "on",
+  "name-length": "off",
+  "no-restricted-tags": [
+    "on",
+    {
+      "tags": [
+        "@watch",
+        "@wip"
+      ]
+    }
+  ],
+  "use-and": "off",
+  "no-duplicate-tags": "on",
+  "no-superfluous-tags": "on",
+  "no-homogenous-tags": "on",
+  "one-space-between-tags": "on",
+  "no-unused-variables": "on",
+  "no-background-only-scenario": "on",
+  "no-empty-background": "on",
+  "scenario-size": [
+    "on",
+    {
+      "steps-length": {
+        "Background": 3,
+        "Scenario": 15
+      }
+    }
+  ]
+}
```

Comment:
Where did you find this?

Reply:
I found it on [this blog](https://medium.com/@virtim/how-we-implemented-gherkin-lint-and-why-you-should-too-6faac532d2a). The config worked OOTB with minor configuration changes.